docker_entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 ADMIN_TOKEN=$(yq e '.admin-token' /data/start9/config.yaml)
-echo "ADMIN_TOKEN=\"${ADMIN_TOKEN}\"" >> /.env
+VW_ADMIN_TOKEN=$(echo -n "$ADMIN_TOKEN" | argon2 "$(openssl rand -base64 32)" -e -id -k 65540 -t 3 -p 4)
+echo "VAULTWARDEN_ADMIN_TOKEN=${WV_ADMIN_TOKEN}" >> /.env
```

Comment:
Is this going to solve the log error that was reported? It seems liked we still want the `ADMIN_TOKEN` env var to exist but be this secure string

Reply:
No, it still complains in the log if you use ADMIN_TOKEN
VAULTWARDEN_ADMIN_TOKEN was newly created to use the new format

Reply:
Ok, but VAULTWARDEN_ADMIN_TOKEN is not being used by the service? I don't really understand its purpose. It seems like we would want to fix error in logs.

Reply:
Yeah, this is weird and either I'm being really dense or there's a bug.  I made an issue:
https://github.com/dani-garcia/vaultwarden/issues/3713

Reply:
Thank you for pointing this out Lucy!

[Fixed by adding single quotes](https://github.com/Start9Labs/vaultwarden-wrapper/pull/40/commits/63b8cc97e8e99bbdb3fb7ea460e5c30f502f0e88) around the token value as suggested by @BlackDex in the discussion topic I created:
https://github.com/dani-garcia/vaultwarden/discussions/3714#discussioncomment-6493249